miniapps/parelag/MultilevelHcurlHdivSolver.cpp

```diff
@@ -32,12 +32,13 @@ void rhsfunc(const Vector &, Vector &);
 
 int main(int argc, char *argv[])
 {
-   // Initialize MPI.
+   // Initialize MPI and HYPRE.
    mpi_session session(argc, argv);
    MPI_Comm comm = MPI_COMM_WORLD;
    int num_ranks, myid;
    MPI_Comm_size(comm, &num_ranks);
    MPI_Comm_rank(comm, &myid);
+   Hypre::Init();
```

Comment:
With this setup, it looks like MPI will be finalized before hypre.

Reply:
To clarify: I think the `mpi_session` class used here is the one defined here: https://github.com/LLNL/parelag/blob/8ce2c98f53be4330f25cb1678b916b6e56971f87/src/utilities/mpiUtils.hpp#L30-L34.

Reply:
I think we should just replace this with our `Mpi::Init()`... any objections?

Reply:
That should be fine. The other option would be to add an explicit `Hypre::Finalize()` at the end, but I think `Mpi::Init` is better.

Reply:
Using `Mpi::Init()` sounds good to me.

Reply:
Addressed in https://github.com/mfem/mfem/pull/2873/commits/d13d86e642c62379eb553dabd2c174718de3666b